packages/-ember-data/tests/unit/store/serializer-for-test.js

```diff
@@ -40,16 +39,17 @@ module('unit/store/serializer_for - DS.Store#serializerFor', function(hooks) {
     );
   });
 
-  test('Calling serializerFor with a type that has not been registered and in an application that does not have an ApplicationSerializer looks up the default Ember Data serializer', function(assert) {
-    assert.ok(
-      store.serializerFor('person') instanceof DS.JSONSerializer,
-      'serializer returned from serializerFor is an instance of DS.JSONSerializer'
-    );
-  });
-
-  testInDebug('Calling serializerFor with a model class should assert', function(assert) {
```

Comment:
Any particular reason we removed this? Is it covered by another test elsewhere? I feel as though I remember a more programmatic test that checked all store APIs for this assertion.

Reply:
🤦‍♂ dumb me...I misread the git log and thought it was some incomplete thing I had added, I'll add it back.